### PR TITLE
Version v1.0.0 release

### PR DESCRIPTION
**Pull request by KevinMenden**
PR for a major release to version v1.0.0.

See changelog for updates.

### CODE REVIEW

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -9,23 +9,22 @@ on:
...
-        python setup.py sdist bdist_wheel
-        twine upload dist/*
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```
Comment (zeehio) [inline .github/workflows/python-publish.yml:22]: I suggest you `python -m pip install --upgrade pip setuptools wheel twine`. I've found some installation errors in other packages due to outdated wheel versions

File: scaden/preprocessing/bulk_simulation.py [hunk 2]
```diff
@@ -287,17 +326,21 @@ def simulate_bulk(
...
+    files = glob.glob(os.path.join(data_path, pattern))
     files = [os.path.basename(x) for x in files]
     datasets = [x.split("_")[0] for x in files]
 
     if len(datasets) == 0:
-        logging.error("No datasets fround! Have you specified the pattern correctly?")
-        sys.exit()
+        logging.error(
+            "No datasets fround! Have you specified the pattern correctly?")
+        sys.exit(1)
```
Comment (zeehio) [inline scaden/preprocessing/bulk_simulation.py:342]: No datasets fround -> found

**Review by zeehio — COMMENTED** [review]
A couple minor suggestions

File: .github/workflows/python-publish.yml (hunk 1 above)
Comment (KevinMenden) [reply .github/workflows/python-publish.yml:22]: Thanks Sergio! Really appreciated :-)

File: scaden/preprocessing/bulk_simulation.py (hunk 2 above)
Comment (KevinMenden) [reply scaden/preprocessing/bulk_simulation.py:342]: Thanks!!

**Pull request merged**